app/static/css/style.css

```diff
@@ -94,3 +89,22 @@ img {
 *[cursor="cell"] > * {
   cursor: cell !important;
 }
+
+#remote-screen[screen="default"] > img {
+  max-height: 50vh;
```

Comment:
Can we roll this back? This makes the screen super tiny and screws up mouse integration:

![tinypilot-mouse-align](https://user-images.githubusercontent.com/7783288/93002377-995e7480-f504-11ea-8b78-8223f082f086.gif)